.github/workflows/build.yml

```diff
@@ -260,15 +308,17 @@ jobs:
       run: signtool sign /f $Env:WINDOWS_CODESIGN_CERTIFICATE_PATH /p $Env:WINDOWS_CODESIGN_CERTIFICATE_PASSWORD *.msi
       working-directory: cmake_build
 
-    - name: "[macOS] Upload build to downloads.mixxx.org"
+    - name: "[macOS/Windows] Upload build to downloads.mixxx.org"
       # skip deploying Ubuntu builds to downloads.mixxx.org because these are deployed to the PPA
-      if: runner.os == 'macOS' && env.DOWNLOADS_HOSTGATOR_DOT_MIXXX_DOT_ORG_KEY_PASSWORD != null
-      run: tools/deploy.sh
+      if: runner.os != 'Linux' && github.event_name == 'push' && env.SSH_PASSWORD != null
+      run: bash tools/deploy.sh ${{ matrix.artifacts_path }}
```

Comment:
cool I don't think it would be that easy to run Bash scripts from Windows

Reply:
It actually wasn't 😅 It took me quite some time to get rsync working. I had to install it using MSYS2 and pacman. Unfortunately, running it from Git bash (`shell: bash`) is not possible, so we need to call bash from powershell. Using MSYS2 directly didn't work either because you can't use something like `shell: ${{ matrix.shell }}` (simply doesn't work).

Reply:
I was thinking you'd have to use WSL.